node-fs-writefile/write-rand.js

```diff
@@ -0,0 +1,11 @@
+const data = new Uint8Array(Buffer.from(Math.random().toString()));
```

Comment:
Just write a string instead of a byte array so the file remembers the UTF-8 encoding. Also it's much cleaner.